tensorflow/buildspec-neuron.yml

```diff
@@ -1,5 +1,5 @@
-  account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
-  region: &REGION <set-$REGION-in-environment>
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
Please correct all the indentation in this file. You could compare to `master` to identify these differences.